app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -66,23 +72,39 @@ public void run() {
             drawScene();
             sleep();
             timePoint++;
+            intervalTimePoint++;
         }
     }
 
     private void receiveLevelDetails() {
         // TODO: Serialize current level data and put it in some container
+
```

Comment:
remove empty line

Reply:
Think it's already done in later commits

Reply:
Still worth doing here.

Reply:
Done